GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -152,25 +149,20 @@ public void executeImport(File file, String serverBase, Map<String, String> crit
                     criteria, questionIdToQuestionDto.values());
 
             List<InstanceData> instanceDataList = new ArrayList<>();
```

Comment:
I wonder if initializing it to empty array is necessary since the line 152 you assing a new value to it. I would just have ` List<InstanceData> instanceDataList = parseSplitSheets(....`

Reply:
Quite right, since the other fork is gone now.